admin/lib/tasks/tailwind.rake

```diff
@@ -3,14 +3,21 @@ namespace :solidus_admin do
     require "tailwindcss-rails"
 
     def config_app_path
-      Rails.root.join("config", "tailwind.config.js.erb")
```

Comment:
optional nit: Oh, I think this removal goes to the previous commit…

Reply:
Thanks! Fixed.